applications/nrf_desktop/src/modules/hid_forward.c

```diff
@@ -183,6 +190,32 @@ static int get_input_report_idx(uint8_t report_id)
 	return -ENOENT;
 }
 
+static size_t get_output_report_idx(uint8_t report_id)
+{
+	for (size_t i = 0; i < ARRAY_SIZE(output_reports); i++) {
+		if (output_reports[i] == report_id) {
+			return i;
+		}
+	}
+	__ASSERT_NO_MSG(false);
+	return 0;
+}
+
+static size_t get_output_report_full_size(uint8_t report_id)
```

Comment:
s/get_output_report_full_size/get_output_report_size

Reply:
In other places in app, we refer to report size as to size of the HID report without sizeof(report_id). Should we keep in consistent here to avoid confusion?

Reply:
that is not true `report_size` in all cases include `sizeof(report_id)`. There might be confusion with REPORT_SIZE_... defines that should be REPORT_DATA_SIZE... but this is another subject.
Discussed offline.